NetKet/Utils/random_utils.hpp

```diff
@@ -40,6 +40,82 @@ void RandomGaussian(Eigen::Matrix<std::complex<double>, Eigen::Dynamic, 1> &par,
         std::complex<double>(distribution(generator), distribution(generator));
   }
 }
+
```

Comment:
Small optional thing: we should maybe include <mpi.h> in this file now and remove it from the samplers, where it is no longer needed (nice!)